kip-0002/fungible-v1.pact

```diff
@@ -0,0 +1,133 @@
+(interface fungible-v1
+
+  " Standard for fungible coins and tokens as specified in KIP-0002. "
+
+   ; ----------------------------------------------------------------------
+   ; Schema
+
+   (defschema account-details
+    @doc "Schema for results of 'account' operation."
+    @model [ (invariant (!= "" sender)) ]
+
+    account:string
+    balance:decimal
+    guard:guard)
+
+   (defschema transfer-schema
+    @doc "Schema for TRANSFER-mgr. Future pact will allow schema inferrence."
+    @model
+      [ (invariant (!= "" sender))
+        (invariant (!= "" receiver))
+        (invariant (!= sender receiver))
+        (invariant (> amount 0.0))
+      ]
+
+    sender:string
+    receiver:string
+    amount:decimal)
+
+   ; ----------------------------------------------------------------------
+   ; Caps
+
+   (defcap TRANSFER:bool
+     ( sender:string
+       receiver:string
+       amount:decimal
+     )
+     @doc " Managed capability sealing AMOUNT for transfer from SENDER to \
+          \ RECEIVER. Permits any number of transfers up to AMOUNT."
+     @managed TRANSFER-mgr
+     )
+
+   (defun TRANSFER-mgr:object{transfer-schema}
+     ( managed:object{transfer-schema}
+       requested:object{transfer-schema}
+     )
+     @doc " Manages TRANSFER capability. REQUESTED should match MANAGED on \
+          \ sender and receiver. AMOUNT should be validated against managed, \
+          \ and linearly managed such that a request for 1.0 amount on a 3.0 \
+          \ managed quantity emits MANAGED with updated amount 2.0."
+     )
+
+   ; ----------------------------------------------------------------------
+   ; Functionality
+
+   (defun transfer-create:string
```

Comment:
This needs a standalone `transfer` function as well

Reply:
That will be in the KIP for `-v2`